README.md

```diff
@@ -1,22 +1,37 @@
-# A Discord Rich Presence addon for Guild Wars 2.
-
-https://gw2rpc.info
-
-## Features:
-* Displaying off:
-   + current map, as well as closes point of interest
-   + current raid or fractal boss
-   + character name, race and profession (elite spec)
-   + commander icon if player is currently commanding a squad
-   + active guild tag (needs API key)
-   + time spent on map
+<p align="center"><img alt="Guild Wars 2 RPC" src="https://media.discordapp.net/attachments/908219512271286282/965798895433170954/GW2.png?width=473&height=473" height="76"></p></img>
```

Comment:
Allthough I really like your new design of the logo, I'd prefer to have the "official" logo there. You can find it hosted on the gw2rpc Server here: https://gw2rpc.info/static/img/logo.png

We could probably create a new logo for future versions if you are skilled :)